boards/xtensa/intel_adsp_cavs15/board.cmake

```diff
@@ -1,5 +1,11 @@
 # SPDX-License-Identifier: Apache-2.0
 
+set(LEAGCY_MODE $ENV{CAVS_LEGACY_MODE})
```

Comment:
LEAGACY -> LEGACY

Reply:
Sorry, thanks so much for correcting it.

Reply:
correct it, thanks!